neo/Prompt/Commands/Wallet.py

```diff
@@ -210,6 +211,33 @@ def command_desc(self):
         return CommandDesc('create_addr', 'create a new wallet address', params=[p1])
 
 
+class CommandWalletClaimGas(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        from_addr_str = None
+        max_coins_per_claim = None
+
+        args = arguments
+        if args:
+            args, from_addr_str = get_from_addr(args)
+
+        if args:
+            max_coins_per_claim = get_arg(args, 0, convert_to_int=True)
+            if not max_coins_per_claim:
+                print("max_coins_to_claim must be an integer")
+                return None, False  # return same as failure in ClaimGas().
+
+        return ClaimGas(PromptData.Wallet, True, from_addr_str, max_coins_per_claim)
+
+    def command_desc(self):
+        p1 = ParameterDesc('max_coins_to_claim', 'maximum number of coins to claim', optional=True)
+        p2 = ParameterDesc('--from-addr', 'source address to claim gas from (if not specified, take first address in wallet)', optional=True)
+        return CommandDesc('claim', 'claim gas', params=[p1, p2])
```

Comment:
Handled with b45e275c80b02e6b40901d8d85ef35ce5bc402d3
It will now be aligned with the longest parameter.

Please tell me if you want 15 as the minimum length (there is no minimum for now).

Reply:
nice change. A minimum would be a good idea to have some consistency if params are short.

Reply:
Minimum added with 8c7c82a640470633c9582158e617d0668d900d41